spec/requests/items_spec.rb

```diff
@@ -53,6 +53,12 @@
   describe "GET /edit" do
     it "renders a successful response" do
       item = Item.create! valid_attributes
+      user = FactoryBot.create(:user, password: "password")
+      sign_in user
+      group = FactoryBot.create(:group)
+      membership = Membership.new(role: 1, user_id: user.id, group: group)
```

Comment:
Should be able to use the actual enum `can_manage` here instead of `1` for better readability.

Reply:
Was meinst du damit genau? Meinst du role: admin, oder was meinst du mit can_manage?

Reply:
Oh, ja meinte `admin`. Genau damit so eine Verwirrung nicht passiert ;)